plugins/LadspaEffect/tap/CMakeLists.txt

```
@@ -1,7 +1,9 @@
 INCLUDE_DIRECTORIES("${CMAKE_SOURCE_DIR}/include")
 FILE(GLOB PLUGIN_SOURCES tap-plugins/*.c)
 LIST(SORT PLUGIN_SOURCES)
-SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -Wno-write-strings -fomit-frame-pointer -fno-strict-aliasing -funroll-loops -ffast-math")
+IF (NOT MSVC)
```

We don't use spaces between command names and parentheses in CMake code, even for control flow commands such as `if`.
```suggestion
IF(NOT MSVC)
```

---

fixed this too